util/unittests/test_html.py

```diff
@@ -110,7 +110,7 @@ def test01_isalistofdictionaries(self):
                         ('Testing isalistofdictionaries output. '
                          'Assert true'))
 
-    def test02_isalistofdictionaries(self):
+    def test_isalistofdictionaries(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_isalistofdictionaries' from line 101

<!-- Comment by [SideCI](https://sideci.com) -->